content/document.js

```diff
@@ -422,54 +464,58 @@ function findAll({
   }
   roots.push(CONTENT_ROOT);
   for (const root of roots) {
-    const searchPattern = [""];
-    if (locales.size) {
-      const localePrefixes = [];
-      for (const [locale, include] of locales) {
-        if (!include) {
-          throw new Error("ability to exclude locales is not supported yet");
+    const api = new fdir()
+      .withFullPaths()
+      .filter((filePath) => {
+        // Note! Due to a bug in fdir, any accidental errors throw here are
+        // swallowed!
+        // See https://github.com/thecodrr/fdir/issues/56
```

Comment:
Looks like we got a reply over in that issue. If withError is not what we want, we might want to nest all of this in our own try/catch so that we can at least get some logs for it.